crates/cosmos/cosmos-chain-components/src/impls/queries/packet_commitment.rs

```diff
@@ -30,14 +30,14 @@ where
         port_id: &Chain::PortId,
         sequence: &Chain::Sequence,
         height: &Chain::Height,
-    ) -> Result<(Chain::PacketCommitment, Chain::CommitmentProof), Chain::Error> {
+    ) -> Result<(Option<Chain::PacketCommitment>, Chain::CommitmentProof), Chain::Error> {
         let commitment_path =
             format!("commitments/ports/{port_id}/channels/{channel_id}/sequences/{sequence}");
 
         let (commitment, proof) = chain
             .query_abci_with_proofs(IBC_QUERY_PATH, commitment_path.as_bytes(), height)
             .await?;
 
-        Ok((commitment, proof))
+        Ok((Some(commitment), proof))
```

Comment:
We do have an issue that the ABCI query is actually returning non-membership proof when the commitment length is empty. But I'll fix that later by making `query_abci_with_proofs` return `Option` as well.